schematic/store/synapse.py

```diff
@@ -51,9 +51,98 @@
 from schematic.exceptions import MissingConfigValueError, AccessCredentialsError
 
 from schematic import CONFIG
+from schematic.utils.general import entity_type_checking
 
 logger = logging.getLogger("Synapse storage")
 
+class ManifestDownload(object):
+    def _download_manifest_to_folder(self, syn, manifest_id):
+        """
+        try downloading a manifest to local cache or a given folder
+        manifest
+        Args:
+            syn: an object of type synapseclient.
+            manifest_id: id of a manifest  
+        Return: 
+            manifest_data: A new Synapse Entity object of the appropriate type
+        """
+        if CONFIG["synapse"]["manifest_folder"]:
```

Comment:
hmm so technically users should download the config file from our repo, and our default config.yml should have "manifest_folder" as a key there (see [here](https://github.com/Sage-Bionetworks/schematic/blob/develop/config.yml#:~:text=manifest_folder%3A%20%27manifests%27)) I am thinking maybe we could deprecate the else statement

Reply:
You can always add a 
```
# TODO potentially deprecate if else statement because...
```

This way you make note of the issues in the code and remember to go back to it.

Reply:
Thanks again. I will add that comment in the code.